cmd/groundstation/main.go

```diff
@@ -14,7 +14,6 @@ import (
 )
 
 func main() {
-	slog.Info("Starting groundstation")
 
```

Comment:
NIT: Why'd this go? I'd like to keep it before we load any config, so you can see that it's started loading the config.

Reply:
No reason in particular, just shuffled things because I wanted to see "started groundstation at port xxxx" but I have no strong feelings about either way.